gap/Parser.gi

```diff
@@ -613,14 +613,50 @@ InstallGlobalFunction( AutoDoc_Parser_ReadFiles,
             current_item!.level := Int( current_command[ 2 ] );
         end,
         @InsertChunk := function()
-            Add( current_item, DocumentationDummy( tree, current_command[ 2 ] ) );
+            local parameters, vars, title, inserted_dummy, position_first_space;
+            parameters := current_command[ 2 ];
+            position_first_space := Position( parameters, ' ' );
+            if position_first_space = fail then
+                title := parameters;
+                vars := [ ];
+            elif IsInt( position_first_space ) then
+                title := parameters{[ 1 .. position_first_space - 1 ]};
+                vars := parameters{[ position_first_space + 1 .. Length( parameters )  ]};
+                vars := SplitString( vars, "," );
+            else
+                ErrorWithPos( Concatenation( "Wrong InsertChunk parameters: ", parameters ) );
+            fi;
+            inserted_dummy := DocumentationDummyInstance( tree, DocumentationDummy( tree, title ) );
+            inserted_dummy!.variable_values := vars;
```

Comment:
It'd feel more natural to me pass `vars` to `DocumentationDummyInstance` (resp. to `InsertChunk` in the bigger change I suggest)